src/CoreWCF.Queue/tests/InMemoryQueue/InMemoryQueueTransportPump.cs

```diff
@@ -70,7 +70,7 @@ public override Task StopPumpAsync(CancellationToken token)
     {
         _cts.Cancel();
         _cts.Dispose();
-        _mres.Wait(token);
+        _mres.Wait(CancellationToken.None);
         _mres.Dispose();
```

Comment:
Curious why you are passing in CancellationToken.None? This is also equivalent of calling Wait() with no parameters.